### DISCUSSION TITLE
Order beatmaps with circles/beat

### DISCUSSION THREAD

**Discussion by LolloDev5123** [ppy/osu#21317] [Q&A]
I currently order my beatmaps with their BPM, but some still have quicker circles to click depending on difficulty, bpm, circle count, etc.
Would there be already a way to order beatmaps based on circles per beat? Or could it be implemented?

**Comment by ghost** [comment]
so basically `total hitobjects / drain time`?

**Reply by LolloDev5123** [reply]
I believe so, it could be an useful statistic for categorizing fast to slow beatmaps, apart from BPM or difficulty
